app/src/main/res/drawable/edit_text_background_shadow.xml

```diff
@@ -2,7 +2,18 @@
 <layer-list xmlns:android="http://schemas.android.com/apk/res/android">
   <item>
     <shape>
-      <corners android:radius="5dp"></corners>
+      <corners android:radius="10dp"></corners>
```

Comment:
You could close `<corner ... />` tags instead of having empty body.

Reply:
Done